packages/shared/src/lib/featureManagement.ts

```diff
@@ -41,6 +42,7 @@ const feature = {
     desktop: 25,
   }),
   pricingIds: new Feature('pricing_ids', {
+    pri_O1jdkx6sakk5kb6p586nejnqam: 'early_adopter',
```

Comment:
This should just be monthly no?
It's not a different cycle and by re-using cycles it makes everything else more inlined.

Reply:
Yes, this is only monthly, I can add the pricing id as value for the experiment if you think is a better idea. Wdyt?